kodex-team-leadera.md

```diff
@@ -0,0 +1,63 @@
+
+### Buď tvrdej, ale fér
+
+Na druhou stranu, když se domluvíte na termínu, vyžaduj jeho splnění, deadliny se musí dodržovat, nebo včas hlásit problém (fail fast).
+
+### Noví lidé, nová krev
+
+Do teamu můžou přijít noví lidé, což je vždycky super, ale může se taky stát, že na to někdo nemá, v Appliftingu to není vždy procházka růžovým sadem. Ty seš ten kdo o tom musí rozhodnout, snaž se to vždycky vnímat nezaujatě, buď fér, ale neber si servítky. Když si nejseš jistej a potřebuješ se poradit, běž a prober to s jiným teamleadem, s klukama z councilu nebo s kýmkoliv o kom si myslíš, že ti může pomoci se správně rozhodnout.
+
+### Plánuj a neber si toho víc, než dokážeš
+
+Známe to, každý už se někdy přecenil, něco blbě nacenil nebo se objevilo něco s čím nepočítal. I přes to se snaž analýze a plánování věnovat fakt velkou pozornost, už to nebude jenom tvůj fail, ale fail celého teamu.
+
+## Vzdělání - Mám zájem o to, aby se můj tým ve svých dovednostech zdokonaloval
+
+### Veď lidi kupředu
+
+Dohlížím, aby lidé v mém týmu měli svého mentora a mohli se tak posouvat ve svých zkušenostech dál. S jejich mentorem konzultuji dovednosti a soft-skills, případně nastavujeme společně další cíle, aby byl náš kolega každým dnem lepším.
+
+## Ego stranou - Buď vzorem, nepovyšuj se
+
+Pomáhej, buď po ruce
+
+Tvým úkolem není lidi v teamu topit s nesmyslných požadavcích, jakmile vidíš, že má někdo s něčím problém, se kterým se neúspěšně trápí už moc dlouho, běž za ním a pomoz mu to vyřešit. Ale zase nechoď moc brzy, nech lidi se ve věcech trochu pohrabat, je třeba ukázat, že v ně věříš! Dělej to zkrátka tak, aby se všichni neustále posouvali ve svých sklillech dál.
+
+### Buď zodpovědný
+
+Vést team s sebou nese příjemné úspěchy, ale i faily. Všichni to známe, když se to s*re, tak se to se*e. V takových chvílích není tvojí povinností házet vinu na ostatní, naopak se máš za team postavit. Show your face, omluv se za celý team a pokus se najít příčinu, kterou s ostatními probereš a uděláte společně všechno pro to, aby se to už příště nestalo.
+
+### Když nevíš, tak nevíš
+
+Neboj se to říct, nikdo není dokonalý. Řekni teamu, že si nejseš jistý, hlavně nemlž. Pak vstaň, zjisti to a deleguj to tam, kam je potřeba.
+
+## Jsi frontmanem Appliftingu - Najdi v sobě diplomata
+
+### Komunikace se zákazníkem
+
+Stane se ti mnohokrát, že bude potřeba něco probrat se zákazníkem. Pokus se odhadnout situaci a vhodně jednat, aby si neudělal ostudu sobě ani Appliftingu. Ale zase se nepřetvařuj, máme svojí identitu. Když myslíme diplomata, nemyslíme populistu, spíš člověka slušného, s citem pro situaci.
+
+### Komunikace vnitřní, teamová
+
+Máš zodpovědnost, tak se snaž zvládat i interní komunikaci. Občas musíš vést standupy, statusy nebo třeba debaty. Je dobré, když tomu dáváš úroveň. Jak vědomostně, tak formálně.
```

Comment:
Myslím znovu, že je to příliš měkké. Byl bych asi trochu ztracený, jak tomu dát úroveň. Co nějaký jednoduchý checklist?

- připrav se předem (chybějící příprava je pohrdání časem ostatních)
- nenech schůzku nekontrolovatelně růst (naplánuj čas předem, přeruš činnosti, které na schůzku nepatří)
- na konci formuluj závěr schůzky (jinak bylo setkání zbytečné)
atd